### PR TITLE
Link PVC volume mode and access mode sections to corresponding sections in PV

### PR DESCRIPTION
**Pull request by verult**
/cc @msau42

### CODE REVIEW

**Comment by sftim** [comment]
/lgtm
Thanks
/sig storage

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -540,11 +540,11 @@ spec:
 
 ### Access Modes
```
Comment (sftim) [inline content/en/docs/concepts/storage/persistent-volumes.md:541]:
```suggestion
### Access Modes {#access-mode-pvc}
```
will set a custom fragment identifier

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -540,11 +540,11 @@ spec:
...
 
-Claims use the same conventions as volumes when requesting storage with specific access modes.
+Claims use [the same conventions as volumes](#access-modes) when requesting storage with specific access modes.
 
 ### Volume Modes
```
Comment (sftim) [inline content/en/docs/concepts/storage/persistent-volumes.md:545]:
```suggestion
### Volume Modes {#volume-mode-pvc}
```
will set a custom fragment identifier (avoiding potential for a collision / confusion)

**Review by sftim — COMMENTED** [review]
An idea for extra tidying (optional)

**Comment by verult** [comment]
This section is linked in k8s API references using the current URL, so while it's cleaner with a more explicit name, we'd need to update API reference doc as well, and it may break other existing links. So I'll keep the header as is.

@sftim would you be able to approve as well? Thanks in advance!

**Comment by sftim** [comment]
> I'll keep the header as is.

OK, fair enough.
/approve

**Pull request merged**